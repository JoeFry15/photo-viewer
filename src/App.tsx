import { useState } from 'react'
import reactLogo from './assets/react.svg'
import { PhotoViewer } from './photoviewer/PhotoViewer';
import { ImageSelector } from './ImageSelector';
import { Menu } from './Menu';

import './App.css'




function App() {

  const [image, setImage] = useState("https://picsum.photos/id/600/600/600.jpg");


  return (
    <div>
      <h1>React Photo Viewer</h1>
      <Menu />
      <PhotoViewer imgUrl={image} />
      <ImageSelector image={image} setState={setImage} />
    </div>
  );
}


export default App
