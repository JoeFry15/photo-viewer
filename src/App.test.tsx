import React from 'react';
import { render } from '@testing-library/react';
import App from './App';

test('renders Hello world text', () => {
    const { getByText } = render(<App />);
    const textElement = getByText(/React/i);
    expect(textElement).toBeInTheDocument();
});