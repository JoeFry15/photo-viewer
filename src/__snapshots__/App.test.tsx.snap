// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="image-wrapper"
>
  <img
    alt="this is a image"
    className="image"
    src="https://picsum.photos/id/600/600/600.jpg"
  />
</div>
`;
